generated/nidcpower/nidcpower/_converters.py

```diff
@@ -275,3 +275,25 @@ def convert_chained_repeated_capability_to_parts(chained_repeated_capability):
     return [','.join(collections.OrderedDict.fromkeys(x)) for x in repeated_capability_lists]
 
 
+# Getter converter for _IsolationState
+___isolation_state_enum_value_to_converted_value_dict = {
+    1128: True,
```

Comment:
you have an enum declared, use that not the hardcoded values

Reply:
My initial rationale of using the enum values instead of the enums themselves for conversion was to avoid the dependency of `_converters.py` on `enums.py`, since some drivers may opt to exclude the codegen of some of these files.

Anyway, I have updated the converter codegen logic to work with enums instead of enum values, and added a logic to conditionally `import enums` if the driver uses enum(s) with converter.